### DISCUSSION TITLE
Add messaging scenarios

### DISCUSSION THREAD

**Discussion by sonwan2020** [Azure-Samples/java-microservices-aca-lab#18] [Ideas]
Message is a key concept to microservices
It is better to add messaging components to the petclinic solution and demo

**Comment by vermegi** [comment]
Yes, add as optional to the end of the lab and based on what's already there in https://azure-samples.github.io/java-microservices-aks-lab/docs/05_lab_messaging/05_openlab_messaging_aks.html

**Reply by vermegi** [reply]
And I guess we want to go for both messaging (service bus) and eventing (event hub), right?

**Reply by sonwan2020** [reply]
thanks, this helps

**Comment by sonwan2020** [comment]
My initial plan is, give messaging or event, they have same topology on the structure.

It depends on the requirement and the efforts
